Mid / Large pop map - NSS Pillar

Peptide90 opened the pull request:
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR <!-- Describe the Pull Request here. What does it change? What other things could this impact? -->
Finally PRing this since it's been sat at 90% for some time waiting on odds and ends.

Everything from the center to the right is based on paradise (box) station (nss cyberiad) however I tried to go for a wide and thin station rather than something sprawling with arms. I do however have some changes I want to make such as adding a proper vault / AI to the central bridge and adding some more to the station above the right side and below engineering. I'll keep evolving this map bigger and denser as we get more content.

Engineering on the far left is intended to have a supermatter engine, until then I've stuck a comfortable spot for an AME to keep things going. No singularities here, ever.

Wanted to get this out there so our players have something new to play for abit when things get cramped on packed and I can work any changes and balance into V1.1 as I add some of the new content in like lights etc.

**Screenshots**
<!-- If applicable, add screenshots to showcase your PR. If your PR is a visual change, add
screenshots or it's liable to be closed by maintainers. -->
![NSSPillar](https://user-images.githubusercontent.com/78795277/151443718-64b27318-4906-47fa-9616-55a1ed2e4bbf.png)
![NSSPillar map](https://user-images.githubusercontent.com/78795277/151697970-1fcfc083-8f7d-4d22-b7b7-77c2eae8bed0.png)

**Changelog**
<!--
Here you can fill out a changelog that will automatically be added to the game when your PR is merged
There are 4 icons for changelog entries: add, remove, tweak, fix. I trust you can figure out the rest.

You can put your name after the :cl: symbol to change the name that shows in the changelog (otherwise it takes your GitHub username)
Like so: :cl: PJB

Generally, only put things in changelogs that players actually care about. Stuff like "Refactored X system, no changes should be visible" shouldn't be on a changelog.

For writing actual entries, don't consider the entry type suffix (e.g. add) to be "part" of the sentence:
bad: - add: a new tool for engineers
good: - add: added a new tool for engineers
-->

:cl:
- add: Added a new large pop map. Welcome to NSS Pillar! Familiar cousin to box, follow the signs so you don't get lost.

moonheart08 reviewed (changes requested):
We don't use old-style windows. Sorry.

Less of a big issue, but maint is pretty sparse.

Peptide90 commented:
- Updated so no more thindows unless it's a single pane and not a square.
- Pulled master so added some of the new exterior lights near exterior airlocks and docks.
- Fixed rotation on docks and changed remaining external airlocks to docks where they should have been.
- Removed instances where lights and signs were on windows.
- Few other misc fixes.

Once I get balance feedback etc from the players I'll expand areas of the station where possible.

Peptide90 commented:
Added a map that's been shaded for most of the different areas. Tried not to get too indepth with it or there would be arrows everywhere but generally all labels are in line with their colour.

Could have made a nice grid template for it but effort.

zlodo commented:
Just an opinion, but I think this map will suffer from one of the two big problems of Reach: virtually no tools are available outside of engineering, other than a few emergency crowbars and a single maintenance closet that will be rushed by assistants at round start.

It means as an assistant the experience is going to get pretty boring (not much opportunity for greytiding indeed if that was the intention, but also no opportunity to help around other than going through the formal process of getting given another job by the HoP), and it also means that other roles can't take the initiative to fix things when things start getting too chaotic to be able to rely solely (or at all) on engineering.

It also means virtually no one other than engineers can hack doors, making them comparatively even more overpowered as traitors than they already are, since they start with a complete toolbelt, whereas other traitors will need to spend 5 tc to get a suspicious toolbox.

Note that both Saltern and Packed go out of their way to make sure that a fair number of assistants (and by extension, everyone) can obtain a complete or mostly complete toolset.

Peptide90 commented:
> Just an opinion, but I think this map will suffer from one of the two big problems of Reach: virtually no tools are available outside of engineering, other than a few emergency crowbars and a single maintenance closet that will be rushed by assistants at round start.
> 
> It means as an assistant the experience is going to get pretty boring (not much opportunity for greytiding indeed if that was the intention, but also no opportunity to help around other than going through the formal process of getting given another job by the HoP), and it also means that other roles can't take the initiative to fix things when things start getting too chaotic to be able to rely solely (or at all) on engineering.
> 
> It also means virtually no one other than engineers can hack doors, making them comparatively even more overpowered as traitors than they already are, since they start with a complete toolbelt, whereas other traitors will need to spend 5 tc to get a suspicious toolbox.
> 
> Note that both Saltern and Packed go out of their way to make sure that a fair number of assistants (and by extension, everyone) can obtain a complete or mostly complete toolset.

Fair comments, the original idea was to keep the map compact but since then I've added the odd maint room here and there and have a lot of changes to make post it getting accepted such as more maint spaces and general areas in the top right corner (essentially squaring off the map). I was hoping that the map would have been merged by now to get other feedback from actual rounds before I made the changes as I essentially have to delete a lot from the center top to the right in order to fit a vault on the bridge area so didn't want to postpone this map even further.

All in all I guess I might have to make the changes now while this is still in limbo. I am working on another map also which is totally different in style and scale to this one and has a load more maint areas in comparison so will use the feedback there also. Thanks,

Peptide90 commented:
Made updates to pillar as I was planning as well as from feedback.

- Moved law offices up to accomodate an AI / Secure upload and storage and a vault on the command deck.
- Moved other offices above bar up to accomodate an NT office, barbers, toilet and vacant office (another tool vendor point) as well as a cryo room / gym for whenever that's a thing.
- Added an observation space to the courtroom to allow for witness testimony.
- Added some escape pod docks around the station, one of them has a mostly kitted out pod attached minus a substation (will work on pod power in future).
- Fixed atmos department so it should run gucci from round start.

moonheart08 commented on Resources/Prototypes/Maps/game.yml:
```diff
@@ -95,6 +95,42 @@
+    prefixCreator: '14'
+  mapPath: Maps/nss_pillar.yml
+  minPlayers: 20
+  overflowJobs: []
```
This needs to contain `Assistant`.
